test/gtest/uct/ib/test_rc.h

```diff
@@ -136,7 +136,7 @@ class test_rc_flow_control : public test_rc {
 
     void test_general(int wnd, int s_thresh, int h_thresh, bool is_fc_enabled);
 
-    void test_pending_grant(int wnd);
+    void test_pending_grant(int wnd, uint64_t *wait_fc_seq = NULL);
```

Comment:
maybe pass value instead of pointer (and special value of 0 or UINT64_MAX means ignore)

Reply:
since this is common RC/DC code, we can't pass value, because we will need cast EP to DC_EP inside common code which doesn't look good

Reply:
ok